### ISSUE TITLE
Image Grid, Simple Masonry: Add ability to override default alt text for image

### ISSUE THREAD

**Issue by AlexGStapleton** [siteorigin/so-widgets-bundle#176]
You should be able to override the default alt text for an image so you can use more contextually relevant alt text.
